scripts/apps/dashboard/world-clock/timezones-all-labels.ts

```diff
@@ -0,0 +1,95 @@
+import {gettext} from 'core/utils';
+
+export const labels = {
```

Comment:
this won't fly as it's on top level, we would need a callback so it can load those translations meanwhile

Reply:
yep, you need to make it a function like `getLabels = () => ({'Africa/Abidjan': gettext('Africa/Abidjan')})` which would only be called after the app has initialized.

If you check the console, you should see many errors that you can't call `gettext` before the translations are loaded.